### PR TITLE
User guide, developer guide, aboutme

### PR DESCRIPTION
**Pull request by beatricetay**
Added more user stories
Added coverall and appveyor
Changed user guide structure

### CODE REVIEW

**Comment by tharshita** [comment]
looks good!

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -301,23 +301,47 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+|`* * *` |user with allergies |exclude allergic ingredients |obtain recipes that are catered to me
 
-|`* * *` |busy student |preselect meals for the week | save time deciding on what to eat each day.
+|`* * *` |user struggling to eat healthier |receive motivation for eating healthy meals |stay motivated on my goal
 
-|`**` |clueless user |ask for suggested recipes | chose a random recipe to start cooking.
+|`* * *` |user who cooks regularly |add my own recipes with the goals they fall under |progress in my goals when I cook my own unique meals
 
-|`*` |student on budget |choose recipes that require lower cost| save money
+|`* * *` |demotivated user |choose a goal for myself |cook more meals and be motivated by my progress
```
Comment (pekchinsiongbrian) [inline docs/DeveloperGuide.adoc]: I think the word "unmotivated" is a more accurate word to use here compared to "demotivated"!

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -301,23 +301,47 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`*` |novice cook |filter recipes by difficulty level |select easier recipes
+
+|`*` |user who is passionate about cooking |share the recipes on social media |show my friends what I have cooked today
+
+|`*` |adventurous user |filter the recipes by cuisine |try a new cuisine every time
+
+|`*` |user who prefers hard-copy materials |save my favourite recipes locally |print them out
+
+|`*` |frequent party host |easily scale up the ingredients needed for a recipe |make the correct amount of food
```
Comment (pekchinsiongbrian) [inline docs/DeveloperGuide.adoc]: I suggest writing "scale up the **_amount_** of ingredients needed..."

File: README.adoc [hunk 3]
```diff
@@ -1,10 +1,10 @@
-= Address Book (Level 3)
+= Healthy Baby
```
Comment (pekchinsiongbrian) [inline README.adoc]: Need to standardize with other instances of "HealthyBaby" (ie. no spacing in between)

**Review by pekchinsiongbrian — COMMENTED** [review]
Thank you for your work Beatrice!

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (beatricetay) [reply docs/DeveloperGuide.adoc]: thanks for reviewing! ill make the changes :)

File: README.adoc (hunk 3 above)
Comment (beatricetay) [reply README.adoc]: thanks for noticing it!

**Pull request merged**